DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -32,6 +33,7 @@ public interface IEventsSheetsService
         Task<DiscordEvent> GetEventAsync(int eventKey);
         Task<DiscordEvent> GetEventFromMessageIdAsync(ulong messageId);
         Task<IEnumerable<DiscordEvent>> ListEventsAsync();
+        Task<IEnumerable<EventResponse>> GetEventResponseOptionsAsync(int eventKey);
```

Comment:
When will this be used? 🙂

Reply:
I'm currently using it internally, and it could be useful in the rest of the bot but it isn't in use at the moment. I've changed it to be a private method in 96305e0 🙂